gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/task/AbstractJKubeTask.java

```diff
@@ -49,6 +50,7 @@
   protected JKubeServiceHub jKubeServiceHub;
   protected static final String DOCKER_BUILD_TIMESTAMP = "docker/build.timestamp";
   protected List<ImageConfiguration> resolvedImages;
+  protected File realResourceDir;
```

Comment:
nit: `realResourceDir`doesn't provide much meaning. Either `applicableResourceDir` or `resolvedResourceDir` (for consistency with the other class variable) make more sense

Reply:
Where is this set?